extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -264,7 +270,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	async addNotebookToPinnedView(bookItem: BookTreeItem): Promise<void> {
 		let notebookPath: string = bookItem.book.contentPath;
 		if (notebookPath) {
-			let notebookDetails: IPinnedNotebook = bookItem.book.root ? { bookPath: bookItem.book.root, notebookPath: notebookPath, title: bookItem.book.title } : { notebookPath: notebookPath };
+			let notebookDetails: IPinnedNotebook = bookItem.book.tableOfContents.sections ? { bookPath: bookItem.book.root, notebookPath: notebookPath, title: bookItem.book.title } : { notebookPath: notebookPath };
```

Comment:
what are you checking for tableOfContents.sections and not using it in the following ternary statement.

Reply:
This is to differentiate an standalone notebook from a book notebook. Retrieving the tree item is different in each case.